recipes-debian/init-system-helpers/init-system-helpers_debian.bb

```diff
@@ -0,0 +1,52 @@
+# base recipe: meta-debian/recipes-debian/init-system-helpers/init-system-helpers_debian.bb
+# base branch: warrior
+# base commit: e6df1b6d8745c81a089fb6de89e3e7a5c5618aaf
+
+SUMMARY = "Helper tools for all init systems"
+DESCRIPTION = "This package contains helper tools that are necessary for switching between\n\
+ the various init systems that Debian contains (e.g. sysvinit, upstart,\n\
+ systemd). An example is deb-systemd-helper, a script that enables systemd unit\n\
+ files without depending on a running systemd.\n\
+ .\n\
+ While this package is maintained by pkg-systemd-maintainers, it is NOT\n\
+ specific to systemd at all. Maintainers of other init systems are welcome to\n\
+ include their helpers in this package."
+
+HOMEPAGE = "https://packages.qa.debian.org/i/init-system-helpers.html"
+
+PR = "r0"
+inherit debian-package
+require recipes-debian/sources/init-system-helpers.inc
+
+LICENSE = "GPLv3+ & BSD"
+LIC_FILES_CHKSUM = "file://script/deb-systemd-invoke;beginline=3;endline=30;md5=42d36293e53d929566aef24e3e3b9ef8"
+
+do_install() {
+	install -d ${D}${bindir}
+	install -d ${D}${datadir}/dh-systemd
+	install -d ${D}${datadir}/perl5/Debian/Debhelper/Sequence
+	install -d ${D}${datadir}/debhelper
+        install -d ${D}${datadir}/man/man8
```

Comment:
@AkitoTakada  I'll ask small changes. please align indent please?

Reply:
We have indented as pointed out.

Reply:
thanks!